src/nvc.c

```diff
@@ -291,10 +252,27 @@ static int elaborate(int argc, char **argv)
          opt_set_str(OPT_DUMP_VCODE, optarg ?: "");
          break;
       case 'c':
-         if (optarg)
-            cover_mask = parse_cover_mask(optarg);
+         if (optarg) {
+            char LOCAL *wc_buf[1] = {
+               xcalloc(32)
+            };
+            opt_separed_t cov_opts[] = {
+               {.opt = "statement",             .mask = COVER_MASK_STMT},
```

Comment:
No need to name the fields explicitly: there's only two fields and they're assigned in order anyway.

```c
{ "statement",    COVER_MASK_STMT },
```